src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1217,6 +1222,27 @@ public static void applySslContext()
         }
     }
 
+    public static void applyCryptoProvider()
+    {
+        try
+        {
+            if (conf.crypto_provider == null)
+                conf.crypto_provider = new ParameterizedClass(NoOpCryptoProvider.class.getName(), null);
+
+            if (conf.crypto_provider.class_name == null)
```

Comment:
@ayushisingh29 if you throw an exception because `class_name` is null, it will be caught in the catch block but there you are referencing `conf.crypto_provider.class_name` and you call `toString()` on it. So while you caught the exception, it may throw `NullPointerException` in that catch block again.